webpack.config.js

```diff
@@ -10,6 +10,7 @@ function getConfig(env) {
       library: 'ReactCrop',
       libraryTarget: 'umd',
       filename: env === 'production' ? 'ReactCrop.min.js' : 'ReactCrop.js',
+      globalObject: 'typeof self !== \'undefined\' ? self : this',
```

Comment:
Can you explain what this is doing?

Reply:
It tells webpack not to use the `window` object explicitly, but to pass `this` instead. https://webpack.js.org/configuration/output/#outputglobalobject